ReactNativeUiLib.podspec

```diff
@@ -0,0 +1,22 @@
+require 'json'
+
+package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
+
+Pod::Spec.new do |s|
+  s.name         = "ReactNativeUiLib"
+  s.version      = package['version']
+  s.summary      = "React Native UI Library"
+
+  s.authors      = "Wix.com"
+  s.homepage     = "https://github.com/wix/react-native-ui-lib"
```

Comment:
Better to add the 'homepage' to the package.json and read it from there

Reply:
Done.